pytest_html/plugin.py

```diff
@@ -159,8 +160,14 @@ def append_extra_html(self, extra, extra_index, test_index):
             href = None
             if extra.get('format') == extras.FORMAT_IMAGE:
                 content = extra.get('content')
-                if content.startswith(('file', 'http')) or \
-                        os.path.isfile(content):
+                try:
+                    is_file = (content.startswith(('file', 'http')) or
+                               isfile(content))
+                except ValueError:
+                    # On Windows, os.path.isfile throws this exception when
+                    # passed a b64 encoded image.
+                    is_file = False
+                if is_file:
```

Comment:
I'm okay with this patch, but the `is_file` variable name is misleading. The variable determines if the content is a URI or a path, and not specifically if it's a file. Perhaps we could use a better variable name, or change the logic to determine if the content is a base64 encoded string only falling back to creating a link when it's not.